<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignaturas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_matematica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Matematica</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/asignaturas.png</normaloff>Iconos/asignaturas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ciencia">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ciencia</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/asignaturas.png</normaloff>Iconos/asignaturas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="b_lenguaje">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lenguaje</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/asignaturas.png</normaloff>Iconos/asignaturas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="b_artistica">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Artistica</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/asignaturas.png</normaloff>Iconos/asignaturas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="botonreturn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Principal</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/descarga.png</normaloff>Iconos/descarga.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
